src/components/SignUp/OffstageRoles.tsx

```diff
@@ -1,22 +1,263 @@
+        value: 'Production Management'
+      },
+      {
+        label: 'Board Op',
+        value: 'Board Op'
+      },
+      {
+        label: 'Run Crew',
+        value: 'Run Crew'
+      }
+    ]
+  },
+  scenicAndProperties: {
+    textHeader: 'Scenic & Properties',
+    checkboxes: [
+      {
+        label: 'Set Design',
+        value: 'Set Design'
+      },
+      {
+        label: 'Technical Direction',
+        value: 'Technical Direction'
+      },
+      {
+        label: 'Properties Designer',
+        value: 'Properties Designer'
+      },
+      {
+        label: 'Scenic Carpentry',
+        value: 'Scenic Carpentry'
+      },
+      {
+        label: 'Charge Artist',
+        value: 'Charge Artist'
+      }
+    ]
+  },
+  lighting: {
+    textHeader: 'Lighting',
+    checkboxes: [
+      {
+        label: 'Lighting Design',
+        value: 'Lighting Design'
+      },
+      {
+        label: 'Projection Design',
+        value: 'Projection Design'
+      },
+      {
+        label: 'Special Effect Design',
+        value: 'Special Effect Design'
+      },
+      {
+        label: 'Master Electrician',
+        value: 'Master Electrician'
+      }
+    ]
+  },
+  sound: {
+    textHeader: 'Sound',
+    checkboxes: [
+      {
+        label: 'Sound Design',
+        value: 'Sound Design'
+      },
+      {
+        label: 'Sound Mixer / Engineer',
+        value: 'Sound Mixer / Engineer'
+      }
+    ]
+  },
+  hairMakeupCostumes: {
+    textHeader: 'Hair, Makeup, Costumes',
+    checkboxes: [
+      {
+        label: 'Costume Design',
+        value: 'Costume Design'
+      },
+      {
+        label: 'Hair & Wig Design',
+        value: 'Hair & Wig Design'
+      },
+      {
+        label: 'Make-up Design',
+        value: 'Make-up Design'
+      }
+    ]
+  }
+};
 
 const OffstageRoles: React.FC<{
   setForm: any;
   formData: any;
 }> = props => {
+  const { formData, setForm } = props;
+  const {
+    offstageRolesGeneral,
+    offstageRolesProduction,
+    offstageRolesScenic,
+    offstageRolesLighting,
+    offstageRolesSound,
+    offstageRolesHairMakeupCostumes
+  } = formData;
+
+
+  const isRoleInRolesSection = (section: string[], role: string) =>
+  section.indexOf(role) > -1;
+
+ 
+  const rolesSectionChange = (chkValue: any, chk: string, fieldName: string, section: string[]) => {
+    let sectionRoles = [...section];
+    let name = fieldName;
+
+    if (chkValue) {
+      if (sectionRoles.indexOf(chk) < 0) {
+        sectionRoles.push(chk);
+      }
+    } else {
+      sectionRoles = sectionRoles.filter(rG => rG !== chk);
+    }
+
+    const target = {
+      name: 'offstageRolesGeneral',
+      value: sectionRoles
+    };
+
+    setForm({ target });
+  };  
+  
   return (
-    <Container>
-      <Row>
-        <Col lg="12">
-          <h1>So, What Do You Like Doing?</h1>
-          <h2>Tell us what positions suit you best</h2>
-        </Col>
-      </Row>
-    </Container>
+    <div>
```

Comment:
You'll want to keep the bootstrap container, row, col, etc. structure here so that we still have our grid:
```
<Container>
  <Row>
    <Col lg="12">
      <Title>SO, WHAT DO YOU LIKE DOING?</Title>
      ...
```